### PR TITLE
Add moonbase zombienet config

### PR DESCRIPTION
**Pull request by imstar15**

### CODE REVIEW

File: zombienets/turing/moonbase.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ws_port = 9946
+rpc_port = 8855
+
+[[parachains]]
+id = 1000
+cumulus_based = true
+chain = "moonbase-local"
+
+[parachains.collator]
+name = "moonbase-col-1"
```
Comment (chrisli30) [inline zombienets/turing/moonbase.toml:32]: Their name is "moonbeam-dev-col-1"

File: zombienets/turing/moonbase.toml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[parachains.collator]
+name = "turing-col-1"
+command = "./target/release/oak-collator"
+ws_port = 9946
+rpc_port = 8855
+
+[[parachains]]
+id = 1000
+cumulus_based = true
+chain = "moonbase-local"
```
Comment (chrisli30) [inline zombienets/turing/moonbase.toml:29]: Their name is "moonbeam-dev"

**Review by chrisli30 — APPROVED** [review]
Looks good, but I left two comments.

File: zombienets/turing/moonbase.toml (hunk 1 above)
Comment (imstar15) [reply zombienets/turing/moonbase.toml]: Fixed

File: zombienets/turing/moonbase.toml (hunk 2 above)
Comment (imstar15) [reply zombienets/turing/moonbase.toml:29]: moonbeam-dev is a single chain for development.
It's not a parachain.

Comment (chrisli30) [reply zombienets/turing/moonbase.toml:29]: oh I see. If that’s the case, let’s revert it to moonbase-col-1.

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: zombienets/turing/moonbase.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+[parachains.collator]
+name = "moonbeam-dev-col-1"
```
Comment (chrisli30) [inline zombienets/turing/moonbase.toml:32]: let’s be consistent, so moonbase-local-col-1

**Review by chrisli30 — APPROVED** [review]
LGTM

**Pull request merged**